### DISCUSSION TITLE
Does screenpipe use gpu or cpu on linux?

### DISCUSSION THREAD

**Discussion by talhaanwarch** [mediar-ai/screenpipe#420] [General]
How to make sure gpu is being used?

**Comment by louis030195** [comment]
@talhaanwarch build with `--features cuda`

**Reply by Froggy232** [reply]
Hi,
Is there a way to make it support AMDGPU, ideally through ROCM?
Thanks you a lot for your software, it's potentially a game changer, I deeply hope it will grow a lot.
Thanks you!

**Reply by louis030195** [reply]
hey i don't think whisper will work right now with ROCM https://github.com/huggingface/candle/issues/346

but you can use deepgram if it uses too much memory or we are looking at alternative like using whisper cpp or MKL maybe

for local AI you can probably leverage amd gpu!

thank you for the kind word :), let me know any feedback you have!

**Comment by louis030195** [comment]
hey @talhaanwarch

closing this as part of our cleanup - screenpipe v2.2.0 is now out with major improvements!

check it out: https://x.com/louis030195/status/2014403576475615312

if this is still relevant or you have questions, feel free to reopen this discussion. thanks for being part of the community!
